software-containers/arcade-casa/version-3.4-4.2/startup.sh

```diff
@@ -1,8 +1,23 @@
 #!/bin/bash
 
-echo "applying leap second patch"
-/arcade/update-data.patch /opt/casa
```

Comment:
I'm probably not up to date on this but just wondering why update-data.patch has been removed...